SW_Flow_lib.c

```diff
@@ -982,7 +982,7 @@ void infiltrate_water_low(double swc[], double drain[], double *drainout, unsign
 
 	unsigned int i;
 	int j;
-	double drainlw = 0.0, swc_avail, drainpot, d[nlyrs], push, kunsat_rel	;
+	double drainlw = 0.0, swc_avail, drainpot, d[OVER_SIZE] = {0}, push, kunsat_rel	;

```

Comment:
Why not use existing constant for this specific purpose of soil layer number, i.e., `MAX_LAYERS`, if we want here a constant array size?